changelogs/fragments/1914-add-sanitization-to-url.yml

```
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - proxmox inventory - added handling of extra trailing slashes in the url (https://github.com/ansible-collections/community.general/pull/1914).
```

```suggestion
  - proxmox inventory - added handling of extra trailing slashes in the URL (https://github.com/ansible-collections/community.general/pull/1914).
```
Sorry, missed one :)

---

Me too ^_^